charts/gardener-extension-shoot-rsyslog-relp/templates/role.yaml

```diff
@@ -30,4 +36,13 @@ rules:
   - gardener-extension-heartbeat
   verbs:
   - update
```

Comment:
You could also allow patch for the leases. It will be anyways required with Gardener v1.117.0. As a best practice, it is nice to allow patch in the places we allow update. Both of the operations are doing the same - `update` updates the whole resource, `patch` - updates the resource partially.